inst/shiny/modules/tab_nca/nca_setup.R

```diff
@@ -324,8 +326,8 @@ nca_setup_server <- function(id, data, adnca_data) { # nolint : TODO: complexity
       updateSelectInput(
         session,
         inputId = "select_dosno",
```

Comment:
suggestion: I think it would be good to rename the widget as well.
```suggestion
        inputId = "select_nca_profile",
```